### PR TITLE
Remove reference to trigger_bucket and trigger_topic parameter

### LINKED ISSUE #9315: Documentation for google_cloudfunctions_function is misleading about trigger_bucket and trigger_topic
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.
* Please do not leave _+1_ or _me too_ comments, they generate extra noise for issue followers and do not help prioritize the request.
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
* If an issue is assigned to the `modular-magician` user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to `hashibot`, a community member has claimed the issue already.

Documentation for [google_cloudfunctions_function](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloudfunctions_function#trigger_http) states that 
```
Cannot be used with trigger_bucket and trigger_topic.
```
But there are no parameters with that name.

### PR DESCRIPTION
**Pull request by muhammadpanji**
Fix #9315

### CODE REVIEW

File: website/docs/r/cloudfunctions_function.html.markdown [hunk 1]
```diff
@@ -123,7 +123,7 @@ Eg. `"nodejs10"`, `"nodejs12"`, `"nodejs14"`, `"python37"`, `"python38"`, `"pyth
 
 * `event_trigger` - (Optional) A source that fires events in response to a condition in another service. Structure is documented below. Cannot be used with `trigger_http`.
 
-* `trigger_http` - (Optional) Boolean variable. Any HTTP request (of a supported type) to the endpoint will trigger function execution. Supported HTTP request types are: POST, PUT, GET, DELETE, and OPTIONS. Endpoint is returned as `https_trigger_url`. Cannot be used with `trigger_bucket` and `trigger_topic`.
+* `trigger_http` - (Optional) Boolean variable. Any HTTP request (of a supported type) to the endpoint will trigger function execution. Supported HTTP request types are: POST, PUT, GET, DELETE, and OPTIONS. Endpoint is returned as `https_trigger_url`. Use `event_trigger` for another trigger options.
```
Comment (megan07) [inline website/docs/r/cloudfunctions_function.html.markdown:126]: Do we want to make this "Cannot be used with `event_trigger`." to stay consistent with the messaging of `event_trigger`?

**Review by megan07 — CHANGES_REQUESTED** [review]
Thanks for fixing this!

File: website/docs/r/cloudfunctions_function.html.markdown (hunk 1 above)
Comment (muhammadpanji) [reply website/docs/r/cloudfunctions_function.html.markdown]: Hi @megan07 ,
Thank you for reviewing this PR. I updated the PR per your suggestion to be consistent with messaging in `event_trigger`. Thank you.

**Review by megan07 — APPROVED** [review]
Thank you!

**Pull request merged**